tests/Routing/RoutingRouteTest.php

```diff
@@ -309,24 +309,6 @@ public function testClassesCanBeInjectedIntoRoutes()
         unset($_SERVER['__test.route_inject']);
     }
 
-    public function testClassesAndVariablesCanBeInjectedIntoRoutes()
```

Comment:
But that test was the only test introduced https://github.com/laravel/framework/pull/17973 that started my issue. What test is now covering the changes in that PR?